ceph/rados/monitor_workflows.py

```diff
@@ -281,3 +282,107 @@ def set_tiebreaker_mon(self, host) -> bool:
             return False
         log.info("Successfully set the new tiebreaker mon on cluster")
         return True
+
+    def connection_score_checks(
+        self, hosts, rados_obj, num_mons, mon_election_obj
```

Comment:
You would not need to pass any of the params that are actually being passed in the method. 

All the passed objects are already part of the class.